### ISSUE TITLE
AddRecaptcha method should be inside Microsoft.Extensions.DependencyInjection

### ISSUE THREAD

**Issue by ultimaweapon** [TimothyMeadows/reCAPTCHA.AspNetCore#78]
In this way, users don't need to import `reCAPTCHA.AspNetCore` namespace. When they referenced `reCAPTCHA.AspNetCore` in their project `AddRecaptcha` will show up in the auto-completion automatically. Other Microsoft projects also use this way ([ref](https://docs.microsoft.com/en-us/dotnet/api/microsoft.extensions.dependencyinjection.entityframeworkservicecollectionextensions?view=efcore-3.1)).

**Comment by mo-esmp** [comment]
Because of `RecaptchaSettings` you have to import `reCAPTCHA.AspNetCore` namespace unless you move `RecaptchaSettings` to `Microsoft.Extensions.DependencyInjection` namespace and I think there is nothing special in moving `AddRecaptcha`  namespace.

**Comment by ultimaweapon** [comment]
Not really. You don't need to reference `RecaptchaSettings` due to you can use type inference on the lambda. [MvcOptions](https://docs.microsoft.com/en-us/dotnet/api/microsoft.aspnetcore.mvc.mvcoptions?view=aspnetcore-3.1) also not inside `Microsoft.Extensions.DependencyInjection`.

**Comment by TimothyMeadows** [comment]
@ultimaweapon i would recommend creating a pull request with this change so it can be tested / approved if you still wish to see this. Thank you for your contributions.

**Comment by ultimaweapon** [comment]
@TimothyMeadows okay will do when I have time.
